pelias-config/pelias.json

```diff
@@ -0,0 +1 @@
+{"logger":{"level":"info","timestamp":false},"esclient":{"apiVersion":"7.5","hosts":[{"host":"elasticsearch"}]},"elasticsearch":{"settings":{"index":{"refresh_interval":"10s","number_of_replicas":"0","number_of_shards":"1"}}},"acceptance-tests":{"endpoints":{"docker":"http://api:4000/v1/"}},"api":{"services":{"pip":{"url":"http://pip:4200"},"libpostal":{"url":"http://libpostal:4400"}},"defaultParameters":{}},"imports":{"adminLookup":{"enabled":false},"csv":{"datapath":"/data/csv","download":["https://s3.amazonaws.com/datatools-dev/bundles/fd64eac2-7a1e-4590-90e9-d1b065673194/23892d66-c546-428d-9bfe-91ed964ff72e/a74679b8-5a9e-4e1b-827b-e3f5cd036e71_new%20custom.csv","https://s3.amazonaws.com/datatools-dev/bundles/fd64eac2-7a1e-4590-90e9-d1b065673194/23892d66-c546-428d-9bfe-91ed964ff72e/0cfa2f37-f4c3-485a-96f6-c1cd96811cac_familyLabels%202%202.csv","https://s3.amazonaws.com/datatools-dev/bundles/fd64eac2-7a1e-4590-90e9-d1b065673194/23892d66-c546-428d-9bfe-91ed964ff72e/0855ede5-1000-457a-89ad-be0dc46cf0b6_familyLabels.csv"],"files":[]},"transit":{"datapath":"/data/transit","feeds":[{"agencyId":"washington park shuttle","agencyName":"washington park shuttle","filename":"washington park shuttle/stops.txt","layerId":"stops","layerName":"address"}]}}}
```

Comment:
Can this be prettified? Also, please remove references to files in datatools-dev bucket.

Reply:
Whoops! Yes it should always have been the clean version but I think this accidentally happened when moving repositories. Restored in dbab857d611c1d6aaf18033fba41217c69a193e2